### ISSUE TITLE
Chronic cough

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6642]
**Preferred term label:**
Chronic cough

**Synonyms**
Chronic coughníng

**Definition (free text, please give PubMed ID)**
Cough lasting longer than eight weeks. PMID: 29094873

**Parent term (use hpo.jax.org/app)**
Cough

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:% 604809, % 608088

**Comment by pnrobinson** [comment]
@eswietlik  @mgriese  -- is this a standard definition for chronic cough, is this a good term?

**Comment by mgriese** [comment]
For adults may be ok, for children:;
Pediatric chronic cough (ie, cough in children aged <15 years) is defined as a daily cough lasting for >4 weeks. This time frame was chosen based on the natural history of URTIs in children and differs from the definition of chronic cough in adults.
PMID: 16428719

**Comment by pnrobinson** [comment]
Thanks, I updated the comment to provide the definition of ``chronic`` for children. The decision was to use the several existing Cough terms together with the HPO modifers terms for Chronic, Subacute, Acute, e.g., https://hpo.jax.org/app/browse/term/HP:0011011
